src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -124,13 +148,30 @@ void* LoadCoreClr(std::string& runtimeDirectory)
         runtimeDirectory.erase(lastSlash);
 
         std::string coreClrDllPath = runtimeDirectory;
```

Comment:
nit: lines 150-160 should be a function - right now we basically copied the code from the above